ArduCopter/Copter.h

```diff
@@ -125,6 +125,11 @@
 #include "afs_copter.h"
 #endif
 
+
+#if USE_WAYBACK == ENABLED
+#include "/AP_WayBack/AP_WayBack.h"
```

Comment:
you should add AP_Wayback to wscript and make.inc, instead of giving a path to the file !